lib/plugin.js

```diff
@@ -17,9 +17,7 @@ function plugin (pluginOptions) {
     digestEncoding: 'base64',
     getAlgorithm,
     algorithmMap: {},
-    getSignatureEncoding: () => {
-      throw new Error('No getSignatureEncoding function provided.')
-    }
+    getSignatureEncoding: () => 'base64'
```

Comment:
I chose to include the default here in plugin.js but I can also pull this out to a separate file to match the pattern of the other default methods. There would just be more boilerplate. Let me know if you want this changed.

Reply:
I think this is fine 👍